lib/package_provider/cached_repository.rb

```diff
@@ -30,40 +35,48 @@ def repo_ready?(path)
     end
 
     def cached_clone(treeish, paths, use_submodules = false)
-      cached_dir = CachedRepository.cache_dir(treeish, paths, use_submodules)
-      return cached_dir if CachedRepository.repo_ready?(cached_dir)
+      @cached_dir = CachedRepository.cache_dir(treeish, paths, use_submodules)
+      if CachedRepository.repo_ready?(@cached_dir)
+        Metriks.meter('packageprovider.repository.cached').mark
```

Comment:
would be nice send metric for particular repo too:
Metriks.meter("packageprovider.repository.#{repo_slug}.cached").mark